src/pybind/mgr/cephadm/migrations.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, mgr: "CephadmOrchestrator"):
         self.migrate()
 
     def set(self, val: int) -> None:
-        self.mgr.set_module_option('migration_current', val)
+        self.mgr.set_module_option('migration_current', str(val))
```

Comment:
Is this required?
```py
# mgr/mgr_module.py
def _set_module_option(self, key, val):
    return self._ceph_set_module_option(self.module_name, key,
                                        None if val is None else str(val))
```

Reply:
If it's for typing compliance, there's a `cast()` helper in `typing` ([it's almost a no-op](https://github.com/python/typing/blob/master/src/typing.py#L1436-L1444), just for mypy to be happy).

Reply:
thanks. I am using `cast()` in the latest revision.

Reply:
sorry, i revisited the code. probably i should use `Any` for val on the callee side. updated accordingly.